web/src/components/core/Attribute.jsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { Label } from "@patternfly/react-core";
+
+import { Description } from "~/components/core";
+import { Icon } from "~/components/layout";
+
+/**
+ * Display a label with additional details. The details are displayed after
+ * clicking the label and the "i" icon indicates available details.
+ * If the label is not defined or is empty it behaves  like a plain label.
+ * @component
+ *
+ * @param {object} description details displayed after clicking the label
+ * @param {object} children the content of the label
+ */
+export default function Attribute ({ description, children }) {
+  if (description) {
```

Comment:
NP: We also have an [`If` component](https://github.com/openSUSE/agama/blob/master/web/src/components/core/If.jsx) for this kind of scenarios where we are just displaying a component or another based on a condition.

Reply:
I tried that it does not look nice with multiple nested components spread over several lines:

```js
  return (
    <If
      condition={description}
      then={
        <Popover showClose={false} bodyContent={description} {...otherProps}>
          <span className="cursor-pointer">{children}</span>
        </Popover>
      } else={
        // none or empty description, just return the children
        children
      }
    />
  );
```

IMHO this looks a bit better:

```js
  if (description) {
    return (
      <Popover showClose={false} bodyContent={description} {...otherProps}>
        <span className="cursor-pointer">{children}</span>
      </Popover>
    );
  }

  // none or empty description, just return the children
  return children;

```